### PR TITLE
Ds terraform intel gcm linux tdx vm

### PR DESCRIPTION
**Pull request by daveshrestha-intel**
Completed Intel TDX VM example for GCP and updated main.tf on both root and example foldesr along with variables on both root  and tdx example folders

### CODE REVIEW

File: examples/gcp-linux-tdx-vm/README.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
+<p align="center">
+  <img src="https://github.com/intel/terraform-intel-gcp-vm/blob/main/images/logo-classicblue-800px.png?raw=true" alt="Intel Logo" width="250"/>
+</p>
+
+# Intel Cloud Optimization Modules for Terraform
```
Comment (kevinbleckmann) [inline examples/gcp-linux-tdx-vm/README.md:5]: Optimized Cloud Modules

File: examples/gcp-linux-tdx-vm/main.tf [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+#                     _ _ __ | |_ ___| |
+#                    | | '_ \| __/ _ \ |
+#                    | | | | | ||  __/ |
+#                    |_|_| |_|\__\___|_|
+# --------------------------------------------------------
+# Provision GCP virtual machine on Intel Xeon 4th Generation Scalable processors (code-named Sapphire Rapids) VM which supports Confidential Computing with Intel Trust Domain Extension (Intel TDX) technology
+# You will need to provide value of the variable project, which is your GCP project id when you do terraform apply
+module "linux_vm" {
+  #source              = "intel/gcp-vm/intel"
+  source              = "../../"
```
Comment (kevinbleckmann) [inline examples/gcp-linux-tdx-vm/main.tf:13]: remove the local source and put the github one back

File: examples/gcp-linux-tdx-vm/variables.tf [hunk 3]
```diff
@@ -0,0 +1,17 @@
+variable "project" {
+  type        = string
+  description = "The ID of the project in which the resource resides."
+default = 551221341017
```
Comment (kevinbleckmann) [inline examples/gcp-linux-tdx-vm/variables.tf:4]: take out our project id, put in 012345678910

File: examples/gcp-linux-tdx-vm/main.tf [hunk 4]
```diff
@@ -0,0 +1,27 @@
+# --------------------------------------------------------
+#                     _       _       _
+#                    (_)     | |     | |
+#                     _ _ __ | |_ ___| |
+#                    | | '_ \| __/ _ \ |
+#                    | | | | | ||  __/ |
+#                    |_|_| |_|\__\___|_|
+# --------------------------------------------------------
+# Provision GCP virtual machine on Intel Xeon 4th Generation Scalable processors (code-named Sapphire Rapids) VM which supports Confidential Computing with Intel Trust Domain Extension (Intel TDX) technology
```
Comment (kevinbleckmann) [inline examples/gcp-linux-tdx-vm/main.tf:9]: 5th Generation Emerald Rapids

File: examples/gcp-linux-tdx-vm/main.tf (hunk 2 above)
Comment (daveshrestha-intel) [reply examples/gcp-linux-tdx-vm/main.tf]: Done

**Review by kevinbleckmann — APPROVED** [review]

**Pull request merged**